### PR TITLE
Milestone 2

### PR DESCRIPTION
**Pull request by TashfeenRao**
- [x] Prepare the directory structure

- [x] Implement the App component

- [x] Implement the BooksList component

- [x] Implement the Book component

- [x] Implement the BooksForm component

- [x] Define the Redux actions and reducers

- [x] Initialize the Redux store

### CODE REVIEW

File: src/components/container/BooksList.js [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+          {store.getState().map(book => (
+            <Book key={book.ID} book={book} />
+          ))}
+
+        </tbody>
+
+      </table>
+    </div>
+  );
+}
```
Comment (Redvanisation) [inline src/components/container/BooksList.js]: You should use `connect` and `mapStateToProps` instead.

File: src/components/container/BooksList.js [hunk 2]
```diff
@@ -0,0 +1,29 @@
+/* eslint-disable import/no-cycle */
```
Comment (Redvanisation) [inline src/components/container/BooksList.js]: Linter rules shouldn't be disabled.

File: src/actions/index.js [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+  const createBook = book => (
+
+    {
+      type: CREATE_BOOK,
+      book,
+    });
+
+  const removeBook = book => ({
+    type: REMOVE_BOOK,
+    book,
```
Comment (Redvanisation) [inline src/actions/index.js]: **[Optional]** The convention in Redux is to give the actions a `type` and a `payload`.

File: src/reducers/index.js [hunk 4]
```diff
@@ -0,0 +1,29 @@
...
+  {
+    ID: Math.floor(Math.random() * 100),
+    title: 'wrong turn',
+    category: 'Horror',
+  },
+];
+
+const store = createStore(BookReducer, initialState);
+
+export default store;
```
Comment (Redvanisation) [inline src/reducers/index.js]: This code should be placed either in another file or in `index.js`.

**Review by Redvanisation — CHANGES_REQUESTED** [review]
# Changes required  ♻️ 

### Hello team  🌟

**Nice work on this project, but the following changes need to be done before moving on ⬇️** 

#### BookList
* [ ] it connects to Redux store and fetches books from the state [screenshot from pathwright](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/react_reducx/bookstore/milestone2/BooksList/ConnetToRedux.png).

* [ ] it uses `mapStateToProps` to fetch books from the `store`

#### reducers index.js

* [ ] It should only combine the app's reducers.






**_Please check and address the comments below_**

### _After making the required changes please submit a new code review request_

### Good luck! 🍀

**Review by Oghenebrume50 — APPROVED** [review]
### APROVED

Hi @TashfeenRao @OlukaDenis kudos for making changes appropriately, this looks good, **this is approved** :+1:  

**RAPHAEL NORIODE** :mask:

**Pull request merged**